packages/plugin-tools/src/editors/ResourcesPicker.tsx

```diff
@@ -189,6 +189,13 @@ export const ResourcesPickerButton: React.FC<ResourcesPickerButtonProps> = props
       : typeof props.value === 'string'
       ? props.value
       : props.value?.options?.get(props.resourceName),
+    updateResourceId(newProps: ResourcesPickerButtonProps) {
```

Comment:
When we are using `useLocalStore`, the values are set for the initial props, but doesn't update for any successive prop changes. Calling this function updates the `resourceId`.